test/integration/resultsPage.js

```diff
@@ -18,6 +18,51 @@ function assertSearchResponse(search, assertions) {
     .end(assertions);
 }
 
+function searchNameRanking(res, expected) {
```

Comment:
Given the 3 functions `searchNameRanking`, `searchDoctorRanking` and `searchAddressRanking` share a lot of code between them can they be refactored so there is a single function that gets passed the variables. Which looks like the locator class e.g. `.results__name` and the if condition to check for the result?